src/angular/schematics/ng-add/migrations/pagination-migration.ts

```diff
@@ -0,0 +1,43 @@
+import { DevkitContext, Migration, ResolvedResource, TargetVersion } from '@angular/cdk/schematics';
+
+import { iterateNodes, MigrationElement, MigrationRecorderRegistry, nodeCheck } from '../../utils';
+
+/**
+ * Migration that updates replaces <sbb-badge> instances with <span sbbBadge="">.
+ */
+export class PaginationMigration extends Migration<null, DevkitContext> {
+  enabled: boolean = this.targetVersion === ('merge' as TargetVersion);
+
+  private _paginations = new MigrationRecorderRegistry(this);
+
+  /** Method that will be called for each Angular template in the program. */
+  visitTemplate(template: ResolvedResource): void {
+    iterateNodes(template.content, (node) => {
+      if (nodeCheck(node).is('sbb-pagination')) {
+        this._paginations.add(template, node);
+      }
+    });
+  }
+
+  postAnalysis() {
+    if (!this._paginations.empty) {
+      this.logger.info('Migrating sbb-pagination usages');
+      this.logger.info(
+        '  See documentation at https://angular.app.sbb.ch/angular/components/pagination for the differences between sbb-pagination and sbb-paginator'
```

Comment:
After thinking about it, we don't document the differences, but just how to use sbb-paginator

```suggestion
        '  See documentation at https://angular.app.sbb.ch/angular/components/pagination on how to configure sbb-paginator'
```